ChatterboxBackend/Chatterbox/Controllers/UserController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Chatterbox.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Chatterbox.Controllers
+{
+    [Route("api/[controller]")]
+    public class UserController:Controller
+    {
+        private readonly ChatterContext _context;
+
+        public UserController(ChatterContext context)
+        {
+            _context = context;
+        }
+
+        [HttpPost("login")]
+        public IActionResult LogMeIn([FromBody] LoginDetails details)
+        {
+            Console.WriteLine("NEW LOG",details.Email);
+            var emailCheck = _context.Users.FirstOrDefault(t => t.Email == details.Email);
+
+
+            if (emailCheck == null)
+            {
+                return new ObjectResult(new { status = "failed",message="No such user", user = "" });
+            }
+
+            var items = _context.Users.FirstOrDefault(t => t.Email == details.Email && t.Password == details.Password);

```

Comment:
Looks like password is being stored plain text. I'm assuming this is a quick and dirty login procedure for the proof-of-concept, but in the future don't do this on an actual project. Hash your passwords, and use existing technology for the security parts of you app. eg. OWIN, Auth0